Week2/homework/js exercises/5-WhoWantsADrink.js

```diff
@@ -0,0 +1,50 @@
+// Declare a variable that holds an empty array, 
+// called drinkTray.
+
+const drinkTray = [];
+const drinkTypes = [" cola", " lemonade", " water"];
+let index = 0;
+for (let i = 0; i < 5; i++) {
+	if(drinkTray.filter(item => item == drinkTypes[index]).length < 2 ) {
```

Comment:
Ow boy! You did something that you guys will only get to experience within the next module. Can you come up with a way of solving this issue without using the `filter` method?